docs/tutorial/fdfitting.rst

```diff
@@ -131,25 +113,23 @@ If we also expect an offset in the dependent parameter, we can simply add an off
     Name            Value  Unit      Fitted      Lower bound    Upper bound
     ------------  -------  --------  --------  -------------  -------------
     DNA/d_offset     0.01  [au]      True               -0.1            0.1
-    DNA/Lp          40     [nm]      True                0            100
-    DNA/Lc          16     [micron]  True                0            inf
-    DNA/St        1500     [pN]      True                0            inf
-    kT               4.11  [pN*nm]   False               0              8
+    DNA/Lp          40     [nm]      True              0.001            100
+    DNA/Lc          16     [micron]  True            0.00034            inf
+    DNA/St        1500     [pN]      True                  1            inf
+    kT               4.11  [pN*nm]   False              3.77              8
     DNA/f_offset     0.01  [pN]      True               -0.1            0.1
 
-From the above example, you can see how easy it is to composite models. Sometimes, models become more complicated. For
-instance, we may have two worm like chain models that we wish to add, and then invert. For the Odijk model, this can be
+Sometimes, models become more complicated. For instance, we may have two worm like chain models that we wish to add, and then invert. For the Odijk model, this can be
```

Comment:
I wonder if it would make sense to discuss when you'd add them by distance (construct) and when you'd add them by force (multiple tethers).

Reply:
It would indeed be good to elaborate on that somewhere. Maybe that would fit in a theory section, rather than a tutorial?

Reply:
Yeah, fair enough. That makes sense. Might still make sense to mention that you add by distances (length)? It is something a user might run into, if they try to add a force to a distance or vice versa.

Reply:
I added an explanation on why methods are added in terms of distance. I do feel that this particular topic deserves an extra theory section, but we can add that when there is time.